content/faq/docker/docker-machine.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Docker Machine
+showMetadata: true
+editable: true
+showToc: true
+---
+# The Docker Machine for AWS.
+
+Docker machine is a tool that lets you install Docker Engine on your virtual host or event Remote host at your cloud provider.
+
+By the way, This content will be talking about AWS cloud providers.
+
+It feels tedious when you want quickly spin up the prototype application using Docker. You might create and install tons of prerequisites. This thing is going to be fine if you have a lot of time. By the way, you don't have much time. What'd you like to do?
```

Comment:
After read a document, I can see main benefit of using machine are:
- Easily setup Docker Engine on EC2 instance
- After you have connect to a machine, you can use a normal Docker command on your computer, e.g. Docker run but it is sent to a Docker engine on EC2

However, I think you still need to find an image or custom images to run your application. 
This makes not totally agree for `You might create and install tons of prerequisites`.

IMHO, if you want more managed container instances, you can use Azure App Service for containers or Beanstalk.
FYI for Azure App Service, it starts from 13USD a month and you can deploy multiple instances.
I agree that this docker-machine can be very useful during a development but do we really need remote docker during a development.
If it is possible, I quite encourage people to use platform as a service over infrastructure as a service.

@dmakeroam  @ponggun are good at this area, would you like to add something?

Reply:
Mostly, I use remote docker during a development also for Kubernetes, but I use SaaS not iaaS